en/streaming-integrator/docs/setup/production-checklist.md

```diff
@@ -0,0 +1,44 @@
+# Production Checklist
+
+Once you download and install WSO2 Streaming Integrator, you may need to update the its default configurations based on your requirements.
```

Comment:
Once you download and install WSO2 Streaming Integrator, you may need to update its default configurations based on your requirements.